README.md

```diff
@@ -272,7 +272,13 @@ If WPA-EAP is used, ieee802_1x settings must be defined in the
 The following options are supported:
 
 - `ssid`: the SSID of the wireless network (required)
-- `key_mgmt`: `wpa-psk` or `wpa-eap` (required)
+- `key_mgmt` (required)
+
+    Any key from following key list:
+    - `wpa-psk`
+    - `wpa-eap`
+    - `owe`
```

Comment:
I prefer list to be ordered/sorted please consider it here.

Reply:
Sorted.